inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx

```diff
@@ -145,6 +158,71 @@ const NodeEditModal: React.FC<NodeEditModalProps> = ({ id, type, clusterId, ...m
           ],
         },
       },
+      {
+        type: 'select',
+        label: i18n.t('pages.Clusters.Node.Agent'),
+        name: 'moduleIdList',
+        rules: [{ required: false }],
```

Comment:
Remove the rule, the default is non-required

Reply:
Done